src/lib.rs

```diff
@@ -210,15 +210,21 @@ stdlib. A `Coroutine` is a generalization of a `Generator`. A `Generator` constr
 resume argument type to `()`, but in a `Coroutine` it can be anything.
 */
 
-#![cfg_attr(feature = "nightly", feature(async_await, async_closure, proc_macro_hygiene))]
+#![cfg_attr(
+    feature = "nightly",
+    feature(async_await, async_closure, proc_macro_hygiene)
+)]
```

Comment:
I added the `proc_macro_hygiene` for the proc-macros to be used on closures.

Reply:
This might not be needed if the closure macro is changed to a function-like macro